src/main/resources/app.conf

```diff
@@ -170,18 +186,22 @@ app {
   permissions {
     retry {
       # the retry strategy to use; possible values are: "never", "once" and "exponential"
-      strategy = "exponential"
+      strategy = ${app.sourcing.retry.strategy}
```

Comment:
this is not a sourcing value, but the retry mechanism for updating permissions; revert to "exponential"